start/pom.xml

```diff
@@ -45,6 +45,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-vfs2</artifactId>
+      <version>2.3</version>
```

Comment:
The place to change this is [here](https://github.com/apache/accumulo/blob/c97342000136c731eaf0990bc9aaa33be05ac53a/pom.xml#L433).  There is a note about issues upgrading to 2.2. However, maybe these issues are fixed in 2.3.  You should try tests and ITs related to this and see if a 2.3 update makes sense.